hummingbird/ml/operator_converters/skl_one_hot_encoder.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import torch
+import numpy as np
+from onnxconverter_common.registration import register_converter
+
+from ._base_operator import BaseOperator
+
+
+class OneHotEncoderString(BaseOperator, torch.nn.Module):
```

Comment:
Do you feel to add some documentation in this class?

Reply:
I added doc to the top of the function. Inside the function, feel free to add more!